pkg/storer/reserve.go

```diff
@@ -276,21 +281,29 @@ func (db *DB) removeChunk(ctx context.Context, address swarm.Address, batchID []
 
 func (db *DB) reserveCleanup(ctx context.Context) error {
 	dur := captureDuration(time.Now())
+	removed := 0
 	defer func() {
 		db.metrics.MethodCallsDuration.WithLabelValues("reserve", "cleanup").Observe(dur())
+		db.metrics.ReserveSize.Set(float64(db.reserve.Size()))
+		db.logger.Info("cleanup finished", "removed", removed, "duration", dur())
```

Comment:
We should also add a metric for this. If this happens a lot in 6 hrs we have a problem.